Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -257,7 +257,8 @@ screen initialState =
                 void $ storeCallBackLocateOnMap push UpdatePickupLocation
                 void $ push $ GoToConfirmingLocationStage
               TryAgain -> do
-                _ <- launchAff $ flowRunner defaultGlobalState $ getEstimate EstimatesTryAgain CheckFlowStatusAction 10 1000.0 push initialState
+                estimatesPolling <- runEffectFn1 getValueFromIdMap "EstimatePolling"
+                when estimatesPolling.shouldPush $ void $ launchAff $ flowRunner defaultGlobalState $ getEstimate EstimatesTryAgain CheckFlowStatusAction 10 1000.0 push initialState estimatesPolling.id
```

Comment:
if you want you can use `TRACKING_ID ` to make sure only one instance is running for polling function in finding estimate and try again case as we do in `driverLocationTracking `